Userland/Utilities/dd.cpp

```diff
@@ -74,13 +74,29 @@ static int handle_size_arguments(size_t& numeric_value, const char* argument)
         return -1;
     }
 
+    unsigned suffix_multiplier = 1;
+    switch(value[value.length()-1]) {
+        case 'k':
+            suffix_multiplier = 1024;
+            value = value.substring(0, value.length()-1);
+            break;
+        case 'M':
+            suffix_multiplier = 1048576;
+            value = value.substring(0, value.length()-1);
+            break;
+        case 'G':
+            suffix_multiplier = 1073741824;
+            value = value.substring(0, value.length()-1);
```

Comment:
We have some handy dandy `constexpr`s you can use instead of having such magical numbers (found in `AK/Types.h`)

```c
constexpr u64 KiB = 1024;
constexpr u64 MiB = KiB * KiB;
constexpr u64 GiB = KiB * KiB * KiB;
```

Reply:
Thank you so much, these are indeed handy! Changes made.